src/core/media/gstreamer/engine.cpp

```diff
@@ -207,14 +208,21 @@ struct gstreamer::Engine::Private
         MH_DEBUG("Got a playbin info message (no action taken): %s", ewi.debug);
     }
 
+    void cleanupAlbumArtFile(const core::ubuntu::media::Track::MetaData& md) 
+    {
+        MetaDataSupport::cleanupTempAlbumArtFile(md);
+    }
+
     void on_tag_available(const gstreamer::Bus::Message::Detail::Tag& tag)
     {
         media::Track::MetaData md;
 
         // We update instead of creating from scratch if same uri
         auto &tuple = track_meta_data.get();
-        if (playbin.uri() == std::get<0>(tuple))
+        if (playbin.uri() == std::get<0>(tuple)) 
```

Comment:
Remove whitspace change.

Reply:
This one tiny whitespace change is perfectly fine in here. No need to block on that alone.